addon/mixins/copyable.js

```diff
@@ -154,14 +154,14 @@ export default Mixin.create({
       ) {
         let value = this.get(name);
 
-        if (Copyable && Copyable.detect(value)) {
+        if ((Copyable && Copyable.detect(value)) || (value && value._isFragment)) {
           // "value" is an Ember.Object using the ember-copy addon
           // (ie. old deprecated Ember.Copyable API - if you use
           // the "Ember Data Model Fragments" addon and "value" is a fragment or
           // if use your own serializer where you deserialize a value to an
           // Ember.Object using this Ember.Copyable API)
           value = value.copy(deep);
-        } else {
+        } else if (value) {
```

Comment:
What is the reason for this change? Previously all values should be run through their transforms even if falsy. For example, you can have a vale of `NaN` that can have a custom transform to be `Not a Number` or `0`, etc...

Reply:
It's to prevent serializing a null fragment since that throws an exception. I can change the check to make sure its not a fragment to prevent reaching this branch in that case.

Reply:
> I can change the check to make sure its not a fragment to prevent reaching this branch in that case.

Yeah, I would rather the check be explicit.